host: codespaces-c89ec1
Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 4
Rules claiming more threads will be scaled down.
Job stats:
job                  count
-----------------  -------
all                      1
run_hefquin_query       10
total                   11

Select jobs to execute...
Execute 4 jobs...

[Mon Mar 17 21:16:29 2025]
localrule run_hefquin_query:
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07a.sparql
    output: output/rdfs/HefQuin/q07a.1.csv, output/rdfs/HefQuin/q07a.1.json, output/rdfs/HefQuin/q07a.1.txt
    jobid: 61
    reason: Missing output files: output/rdfs/HefQuin/q07a.1.csv
    wildcards: workload=rdfs, query=q07a, run=1
    priority: 50
    resources: tmpdir=/tmp


[Mon Mar 17 21:16:29 2025]
localrule run_hefquin_query:
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07j.sparql
    output: output/rdfs/HefQuin/q07j.1.csv, output/rdfs/HefQuin/q07j.1.json, output/rdfs/HefQuin/q07j.1.txt
    jobid: 70
    reason: Missing output files: output/rdfs/HefQuin/q07j.1.csv
    wildcards: workload=rdfs, query=q07j, run=1
    priority: 50
    resources: tmpdir=/tmp


[Mon Mar 17 21:16:29 2025]
localrule run_hefquin_query:
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07d.sparql
    output: output/rdfs/HefQuin/q07d.1.csv, output/rdfs/HefQuin/q07d.1.json, output/rdfs/HefQuin/q07d.1.txt
    jobid: 64
    reason: Missing output files: output/rdfs/HefQuin/q07d.1.csv
    wildcards: workload=rdfs, query=q07d, run=1
    priority: 50
    resources: tmpdir=/tmp


[Mon Mar 17 21:16:29 2025]
localrule run_hefquin_query:
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07g.sparql
    output: output/rdfs/HefQuin/q07g.1.csv, output/rdfs/HefQuin/q07g.1.json, output/rdfs/HefQuin/q07g.1.txt
    jobid: 67
    reason: Missing output files: output/rdfs/HefQuin/q07g.1.csv
    wildcards: workload=rdfs, query=q07g, run=1
    priority: 50
    resources: tmpdir=/tmp

[Mon Mar 17 21:16:36 2025]
Finished job 61.
1 of 11 steps (9%) done
Select jobs to execute...
Execute 1 jobs...

[Mon Mar 17 21:16:36 2025]
localrule run_hefquin_query:
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07c.sparql
    output: output/rdfs/HefQuin/q07c.1.csv, output/rdfs/HefQuin/q07c.1.json, output/rdfs/HefQuin/q07c.1.txt
    jobid: 63
    reason: Missing output files: output/rdfs/HefQuin/q07c.1.csv
    wildcards: workload=rdfs, query=q07c, run=1
    priority: 50
    resources: tmpdir=/tmp

[Mon Mar 17 21:16:36 2025]
Finished job 67.
2 of 11 steps (18%) done
Select jobs to execute...
Execute 1 jobs...

[Mon Mar 17 21:16:36 2025]
localrule run_hefquin_query:
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07i.sparql
    output: output/rdfs/HefQuin/q07i.1.csv, output/rdfs/HefQuin/q07i.1.json, output/rdfs/HefQuin/q07i.1.txt
    jobid: 69
    reason: Missing output files: output/rdfs/HefQuin/q07i.1.csv
    wildcards: workload=rdfs, query=q07i, run=1
    priority: 50
    resources: tmpdir=/tmp

[Mon Mar 17 21:16:36 2025]
Finished job 64.
3 of 11 steps (27%) done
Select jobs to execute...
Execute 1 jobs...

[Mon Mar 17 21:16:36 2025]
localrule run_hefquin_query:
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07f.sparql
    output: output/rdfs/HefQuin/q07f.1.csv, output/rdfs/HefQuin/q07f.1.json, output/rdfs/HefQuin/q07f.1.txt
    jobid: 66
    reason: Missing output files: output/rdfs/HefQuin/q07f.1.csv
    wildcards: workload=rdfs, query=q07f, run=1
    priority: 50
    resources: tmpdir=/tmp

[Mon Mar 17 21:16:36 2025]
Finished job 70.
4 of 11 steps (36%) done
Select jobs to execute...
Execute 1 jobs...

[Mon Mar 17 21:16:36 2025]
localrule run_hefquin_query:
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07h.sparql
    output: output/rdfs/HefQuin/q07h.1.csv, output/rdfs/HefQuin/q07h.1.json, output/rdfs/HefQuin/q07h.1.txt
    jobid: 68
    reason: Missing output files: output/rdfs/HefQuin/q07h.1.csv
    wildcards: workload=rdfs, query=q07h, run=1
    priority: 50
    resources: tmpdir=/tmp

Terminating processes on user request, this might take some time.
Traceback (most recent call last):

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 232, in spawn_job
    subprocess.check_call(cmd, shell=True)

  File "/usr/local/python/3.12.1/lib/python3.12/subprocess.py", line 413, in check_call
    raise CalledProcessError(retcode, cmd)

subprocess.CalledProcessError: Command 'cd /workspaces/P-TER && /usr/local/python/3.12.1/bin/python3 -m snakemake --snakefile '/workspaces/P-TER/Snakefile' --target-jobs 'run_hefquin_query:workload=rdfs,query=q07f,run=1' --allowed-rules run_hefquin_query --cores 4 --attempt 1 --force-use-threads  --force --target-files-omit-workdir-adjustment --keep-storage-local-copies --max-inventory-time 0 --nocolor --notemp --no-hooks --nolock --ignore-incomplete --rerun-triggers input mtime software-env params code --conda-frontend 'conda' --shared-fs-usage sources software-deployment input-output source-cache persistence storage-local-copies --wrapper-prefix 'https://github.com/snakemake/snakemake-wrappers/raw/' --latency-wait 5 --scheduler 'greedy' --local-storage-prefix .snakemake/storage --scheduler-solver-path '/usr/local/python/3.12.1/bin' --default-resources base64//dG1wZGlyPXN5c3RlbV90bXBkaXI= --quiet progress rules --mode 'subprocess' --local-groupid 'local'' died with <Signals.SIGINT: 2>.


During handling of the above exception, another exception occurred:


Traceback (most recent call last):

  File "/usr/local/python/3.12.1/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 247, in cached_or_run
    run_func(*args)

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 234, in spawn_job
    raise SpawnedJobError()

snakemake.exceptions.SpawnedJobError

[Mon Mar 17 21:16:42 2025]
Error in rule run_hefquin_query:
    jobid: 66
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07f.sparql
    output: output/rdfs/HefQuin/q07f.1.csv, output/rdfs/HefQuin/q07f.1.json, output/rdfs/HefQuin/q07f.1.txt

Traceback (most recent call last):

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 232, in spawn_job
    subprocess.check_call(cmd, shell=True)

  File "/usr/local/python/3.12.1/lib/python3.12/subprocess.py", line 413, in check_call
    raise CalledProcessError(retcode, cmd)

subprocess.CalledProcessError: Command 'cd /workspaces/P-TER && /usr/local/python/3.12.1/bin/python3 -m snakemake --snakefile '/workspaces/P-TER/Snakefile' --target-jobs 'run_hefquin_query:workload=rdfs,query=q07c,run=1' --allowed-rules run_hefquin_query --cores 4 --attempt 1 --force-use-threads  --force --target-files-omit-workdir-adjustment --keep-storage-local-copies --max-inventory-time 0 --nocolor --notemp --no-hooks --nolock --ignore-incomplete --rerun-triggers input mtime software-env params code --conda-frontend 'conda' --shared-fs-usage sources software-deployment input-output source-cache persistence storage-local-copies --wrapper-prefix 'https://github.com/snakemake/snakemake-wrappers/raw/' --latency-wait 5 --scheduler 'greedy' --local-storage-prefix .snakemake/storage --scheduler-solver-path '/usr/local/python/3.12.1/bin' --default-resources base64//dG1wZGlyPXN5c3RlbV90bXBkaXI= --quiet progress rules --mode 'subprocess' --local-groupid 'local'' died with <Signals.SIGINT: 2>.


During handling of the above exception, another exception occurred:


Traceback (most recent call last):

  File "/usr/local/python/3.12.1/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 247, in cached_or_run
    run_func(*args)

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 234, in spawn_job
    raise SpawnedJobError()

snakemake.exceptions.SpawnedJobError

[Mon Mar 17 21:16:42 2025]
Error in rule run_hefquin_query:
    jobid: 63
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07c.sparql
    output: output/rdfs/HefQuin/q07c.1.csv, output/rdfs/HefQuin/q07c.1.json, output/rdfs/HefQuin/q07c.1.txt

Traceback (most recent call last):

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 232, in spawn_job
    subprocess.check_call(cmd, shell=True)

  File "/usr/local/python/3.12.1/lib/python3.12/subprocess.py", line 413, in check_call
    raise CalledProcessError(retcode, cmd)

subprocess.CalledProcessError: Command 'cd /workspaces/P-TER && /usr/local/python/3.12.1/bin/python3 -m snakemake --snakefile '/workspaces/P-TER/Snakefile' --target-jobs 'run_hefquin_query:workload=rdfs,query=q07h,run=1' --allowed-rules run_hefquin_query --cores 4 --attempt 1 --force-use-threads  --force --target-files-omit-workdir-adjustment --keep-storage-local-copies --max-inventory-time 0 --nocolor --notemp --no-hooks --nolock --ignore-incomplete --rerun-triggers input mtime software-env params code --conda-frontend 'conda' --shared-fs-usage sources software-deployment input-output source-cache persistence storage-local-copies --wrapper-prefix 'https://github.com/snakemake/snakemake-wrappers/raw/' --latency-wait 5 --scheduler 'greedy' --local-storage-prefix .snakemake/storage --scheduler-solver-path '/usr/local/python/3.12.1/bin' --default-resources base64//dG1wZGlyPXN5c3RlbV90bXBkaXI= --quiet progress rules --mode 'subprocess' --local-groupid 'local'' died with <Signals.SIGINT: 2>.


During handling of the above exception, another exception occurred:


Traceback (most recent call last):

  File "/usr/local/python/3.12.1/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 247, in cached_or_run
    run_func(*args)

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 234, in spawn_job
    raise SpawnedJobError()

snakemake.exceptions.SpawnedJobError

[Mon Mar 17 21:16:42 2025]
Error in rule run_hefquin_query:
    jobid: 68
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07h.sparql
    output: output/rdfs/HefQuin/q07h.1.csv, output/rdfs/HefQuin/q07h.1.json, output/rdfs/HefQuin/q07h.1.txt

Traceback (most recent call last):

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 232, in spawn_job
    subprocess.check_call(cmd, shell=True)

  File "/usr/local/python/3.12.1/lib/python3.12/subprocess.py", line 413, in check_call
    raise CalledProcessError(retcode, cmd)

subprocess.CalledProcessError: Command 'cd /workspaces/P-TER && /usr/local/python/3.12.1/bin/python3 -m snakemake --snakefile '/workspaces/P-TER/Snakefile' --target-jobs 'run_hefquin_query:workload=rdfs,query=q07i,run=1' --allowed-rules run_hefquin_query --cores 4 --attempt 1 --force-use-threads  --force --target-files-omit-workdir-adjustment --keep-storage-local-copies --max-inventory-time 0 --nocolor --notemp --no-hooks --nolock --ignore-incomplete --rerun-triggers input mtime software-env params code --conda-frontend 'conda' --shared-fs-usage sources software-deployment input-output source-cache persistence storage-local-copies --wrapper-prefix 'https://github.com/snakemake/snakemake-wrappers/raw/' --latency-wait 5 --scheduler 'greedy' --local-storage-prefix .snakemake/storage --scheduler-solver-path '/usr/local/python/3.12.1/bin' --default-resources base64//dG1wZGlyPXN5c3RlbV90bXBkaXI= --quiet progress rules --mode 'subprocess' --local-groupid 'local'' died with <Signals.SIGINT: 2>.


During handling of the above exception, another exception occurred:


Traceback (most recent call last):

  File "/usr/local/python/3.12.1/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 247, in cached_or_run
    run_func(*args)

  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/snakemake/executors/local.py", line 234, in spawn_job
    raise SpawnedJobError()

snakemake.exceptions.SpawnedJobError

[Mon Mar 17 21:16:42 2025]
Error in rule run_hefquin_query:
    jobid: 69
    input: /workspaces/P-TER/virtuoso-opensource-7.2.7, /workspaces/P-TER/virtuoso-opensource-7.2.7/var/lib/virtuoso/db/fedup.ini, queries/q07i.sparql
    output: output/rdfs/HefQuin/q07i.1.csv, output/rdfs/HefQuin/q07i.1.json, output/rdfs/HefQuin/q07i.1.txt

Complete log: .snakemake/log/2025-03-17T211629.843519.snakemake.log
WorkflowError:
At least one job did not complete successfully.
